eppo_core/src/event_ingestion/event_delivery.rs

```diff
@@ -7,20 +10,48 @@ use std::collections::HashSet;
 use url::Url;
 use uuid::Uuid;
 
+const MAX_EVENT_SERIALIZED_LENGTH: usize = 4096;
+
 #[derive(Clone)]
 pub(super) struct EventDelivery {
     sdk_key: Str,
     ingestion_url: Url,
     client: reqwest::Client,
 }
 
-#[derive(serde::Deserialize)]
+#[derive(Debug, PartialEq)]
 pub(super) struct EventDeliveryResponse {
```

Comment:
My understanding is that this is no longer a server response but something else

Reply:
yup, i can rename for clarity